packages/sdk-codegen/CHANGELOG.md

```diff
@@ -5,9 +5,34 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [21.1.0](https://www.github.com/looker-open-source/sdk-codegen/compare/sdk-codegen-v21.0.10...sdk-codegen-v21.1.0) (2021-04-02)
+
+
+### ⚠ BREAKING CHANGES
+
+* Update the python SDK to expose these methods on the Looker API.
+
```

Comment:
I'm tempted to go into this branch and manually delete this. I should have been more careful with separating PRs/commits as this entry really doesn't make sense in this changelog

Reply:
I agree, but I think it's fine either way.